api/spec/requests/api/v1/project_collections_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "swagger_helper"
+
+RSpec.describe "Project Collections", type: :request do
+
+  included_relationships = [
+    :subjects,
+    :collection_projects,
+    :"collection_projects.project"
```

Comment:
I have not seen this `:"collection_projects.project"`syntax before as an included relationship. Does this mean that the `collection_projects` associated `projects` should also be serialized in the response?

Reply:
Yes, that is correct.